package.json

```diff
@@ -5,15 +5,19 @@
     "build": "bundle exec jekyll build",
     "clean": "bundle exec jekyll clean",
     "reset": "npx rimraf Gemfile.lock package-lock.json LICENSE.md CONTRIBUTING.md degit.json .git",
-    "start": "bundle exec jekyll serve"
+    "start": "bundle exec jekyll serve",
+    "preinstall": "npx npm-force-resolutions"
```

Comment:
@mheadd I'm not familiar with this package but looks cool.  The only thing that I would be concerned about is the repository dependencies being out of sync from the actual deploy. From their docs: `Then add npm-force-resolutions to the preinstall script so that it patches the package-lock file before every npm install you run` and we run `npm ci` to install the JS dependencies so this may cause the discrepancy between source control and deployed. Can this package be run just once to update the package-lock.json and not use it in a preinstall script?

Reply:
Worth a try

Reply:
OK, that seemed to work. I might kill this PR and rerun the steps so that the only thing that is committed is the modified package-lock file.